README.md

```diff
@@ -117,6 +117,8 @@ Under the hood it uses [Karma test runner](http://karma-runner.github.io/) to al
 
 By default Sagui uses [PhantomJS](http://phantomjs.org/) to run the tests headlessly. To **speed up installing the dependencies** (`npm install`) it is advisable to have PhantomJS [installed globally](https://github.com/Medium/phantomjs#using-phantomjs-from-disk) in the machine.
 
```

Comment:
What do you think about adding a header like `#### Debugging`, so that people can more easily find it?